src/core/Akka/Configuration/Hocon/HoconSubstitution.cs

```diff
@@ -104,20 +104,6 @@ public HoconObject GetObject()
         {
             return ResolvedValue.GetObject();
         }
-
-        #region Implicit operators
-
-        /// <summary>
-        /// Performs an implicit conversion from <see cref="HoconSubstitution" /> to <see cref="HoconObject" />.
-        /// </summary>
-        /// <param name="substitution">The HOCON object that contains the substitution.</param>
-        /// <returns>The HOCON object contained in the substitution.</returns>
-        public static implicit operator HoconObject(HoconSubstitution substitution)
-        {
-            return substitution.GetObject();
-        }
-
-        #endregion
```

Comment:
I have no idea why we had this strange implicit operator support.  casting from a hocon substitution to a hocon object implicitly.
It was not used, and it was harmful as the cast could fail as not all substitutions are objects.